doc/Architecture/arc-ss_x-road_security_server_architecture.md

```diff
@@ -6,8 +6,8 @@
 
 **Technical Specification** <!-- omit in toc -->
 
-Version: 1.9  
-25.10.2019
+Version: 1.10
```

Comment:
Didn't realize that those 2 extra spaces were meaningful and added the line break. That's a bit hard to miss syntax, I wonder if

```
Version: 1.10<br/>
10.09.2020

```
would more clearly indicate the intention for future editors?

Reply:
Fixed using the `<br/>` method now.

Reply:
The double space method seems to be pretty controversial feature in Markdown. One good reason to not use double space is probably code formatting that might automatically clean trailing spaces 🤔. It sounds good to me to use `<br/>`. Though it would be nice to use the same format across all docs.